lints/lint_rsa_mod_not_odd.go

```diff
@@ -27,9 +27,12 @@ func (l *rsaParsedTestsKeyModOdd) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rsaParsedTestsKeyModOdd) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	theKey := c.PublicKey.(*rsa.PublicKey)
+	key, ok := c.PublicKey.(*rsa.PublicKey)
+	if !ok {
+		return ResultStruct{Result: Error}, nil
+	}
 	z := big.NewInt(0)
```

Comment:
It look like `z := big.NewInt(0)` and `big.NewInt(1)` should both be called in init.